utils/python/CIME/case_setup.py

```diff
@@ -84,7 +84,7 @@ def case_setup(caseroot, clean=False, test_mode=False, reset=False):
 
     cimeroot = os.environ["CIMEROOT"]
 
-    case = Case()
+    case = Case(caseroot)
```

Comment:
Due to the chdir above, this argument is not necessary. In fact, if caseroot happens to be relative, this change breaks the code.